editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5504,21 +5504,28 @@ void CanvasItemEditorViewport::_create_preview(const Vector<String> &files) cons
 		Ref<Texture2D> texture = Ref<Texture2D>(Object::cast_to<Texture2D>(*res));
 		Ref<PackedScene> scene = Ref<PackedScene>(Object::cast_to<PackedScene>(*res));
 		if (texture != nullptr || scene != nullptr) {
+			bool root_node_selected = EditorNode::get_singleton()->get_editor_selection()->is_selected(EditorNode::get_singleton()->get_edited_scene());
+			String desc = "Drag and drop to add as child of current scene's root node.\nHold Ctrl when dropping to add as child of selected node.\n";
+			if (!root_node_selected) {
+				desc += "Hold Shift when dropping to add as sibling of selected node.\n";
+			}
```

Comment:
Indeed as @RedMser mentioned, `TTR()` should be used with string literals, it won't work to do `TTR(desc)`.

So this should likely be split line by line, even though this will lose some context for the translators (but they should still be able to infer it as the strings will be grouped together). And then to avoid any mistake the newlines should be kept out of the translation strings, so we don't rely on translators keeping them at the end of each line.

As the last line differs based on the conditions, the newlines should be added at the start of any `desc +=` addition, instead of the end of the previous message.

```suggestion
			String desc = TTR("Drag and drop to add as child of current scene's root node.") + "\n" + TTR("Hold Ctrl when dropping to add as child of selected node.");
			if (!root_node_selected) {
				desc += "\n" + TTR("Hold Shift when dropping to add as sibling of selected node.");
			}
```